components/lib/Badge/Badge.module.scss

```diff
@@ -1,25 +1,13 @@
 .badge {
-    $size: 1.3rem;
-    width: $size;
-    height: $size;
-    font-size: 0.7rem;
-    border-radius: 100rem;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    position: absolute;
-    top: 2px;
-    right: 2px;
-
-    &--blue {
-        background-color: var(--blue);
-        color: white;
-    }
-    &--outline-blue {
-        background-color: white;
-        color: var(--blue);
-    }
-    &--white {
-        background-color: white;
-    }
-}
\ No newline at end of file
+  $size: 1.3rem;
```

Comment:
<img width="1440" alt="Screenshot 2023-04-06 at 22 46 11" src="https://user-images.githubusercontent.com/55176773/230456230-9441af63-a0bc-4665-a488-63a4e65846fb.png">
you changed badge styling and it broke the UI